src/graph.py

```diff
@@ -540,68 +552,70 @@ def randomizations(self, noisy_edges_reps, random_terminals_reps):
                 #######          Grid Search          #######
     ###########################################################################
 
-    def _eval_pcsf(self, params):
-        """
-        Convenience methods which sets parameters and performs PCSF
+    def _eval_PCSF_runs(self, params):
         """
-        self._reset_hyperparameters(params=params)
-        paramstring = 'G_'+str(self.params.g)+'_B_'+str(self.params.b)+'_W_'+str(self.params.w)
-        print(paramstring)
-        return (paramstring, self.pcsf())
+        Convenience method which sets parameters and performs PCSF randomizations.
 
+        Arguments: 
+            params (dict): params with which to run the program
 
-    def _grid_pcsf(self, prize_file, Gs, Bs, Ws):
+        Returns: 
+            str: Parameter values in string format 
+            networkx.Graph: forest
+            networkx.Graph: augmented_forest
         """
-        Internal function which executes pcsf at every point in a parameter grid.
-        Subroutine of `grid_search`.
-
-        Arguments:
-            prize_file (str): filepath
-            Gs (list): Values of gamma
-            Bs (list): Values of beta
-            Ws (list): Values of omega
 
-        Returns:
-            list: list of tuples of vertex indices and edge indices
-        """
+        self._reset_hyperparameters(params=params)
+        paramstring = "W_{:04.2f}_B_{:04.2f}_G_{:d}".format(self.params.w, self.params.b, self.params.g)
 
-        self.prepare_prizes(prize_file)
-        parameter_permutations = [{'g':g,'b':b,'w':w} for (g, b, w) in product(Gs, Bs, Ws)]
-        results = list(map(self._eval_pcsf, parameter_permutations))
+        if params["noisy_edge_reps"] + params["random_terminals_reps"] == 0:
+            logger.info("Single PCSF runs for " + paramstring)
```

Comment:
why is "runs" plural?

Reply:
f54fb926c708d3848b45c34130ec1b8ef5d192c4